Maple2.Server.World/Migrations/20221105211352_ChatSticker.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Maple2.Server.World.Migrations
+{
+    public partial class ChatSticker : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.RenameColumn(
+                name: "Stamps",
+                table: "character-unlock",
+                newName: "StickerSets");
+
+            migrationBuilder.AddColumn<string>(
+                name: "FavoriteStickers",
+                table: "character-config",
+                type: "json",
+                nullable: false)
```

Comment:
If you're not letting this be nullable, you must specify a `defaultValue: "{}"` otherwise this will become invalid for existing rows.

Reply:
Is there a proper way to set this? Or do I edit this generated migration file?

Reply:
i think you just have to manually set it, that's how ive done it before